tests/stack/polymorphic.wast

```diff
@@ -0,0 +1,43 @@
+// drop
+
+i32.const 15
+drop
+#assertStack .Stack "drop i32"
+
+i64.const 15
+drop
+#assertStack .Stack "drop i64"
+
+f32.const 15.0
+drop
+#assertStack .Stack "drop f32"
+
+f64.const 15.0
+drop
+#assertStack .Stack "drop f64"
+
+// select
+
+i32.const -1
+i32.const 1
+i32.const 1
+select
+#assertTopStack < i32 > 1 "select i32 true"
```

Comment:
Where are the `true`/`false` literals coming from?

Reply:
`"true"` here is just part of the string. The `#assertTopStack` is an extra thing I put in for the testing harness, the string (which is last/second argument) is just for naming the test.

In this case, it should select the "true" branch of the `select`, which means keep the top element of the stack instead of the second element.